demos/index.html

```diff
@@ -88,7 +89,11 @@
                             gradeNames: ["gpii.firstDiscovery.auxSchema"]
                         },
                         prefsEditor: {
-                            prefsEditorType: "gpii.firstDiscovery.firstDiscoveryEditor"
+                            store: {
```

Comment:
Shouldn't this be the default setting for the first discovery tool though. If that's the case, I think it should be configured in the component itself rather than the demo. If this is a demo specific setting, it's fine to stay here.